packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -3060,6 +3060,17 @@ User permissions are only effective with respect to the Group(s) to which the ad
 | `okta.apps.assignment.manage`           | Allows the admin to only manage assignment operations of an App in your Okta org                                                                     | All Apps, All apps of specific type, a specific App |
 | `okta.profilesource.import.run`         | Allows the admin to run imports for apps with a profile source, such as HRaaS and AD/LDAP apps. Admins with this permission can create users through the import. | All Apps, All apps of specific type, a specific App |
 
+#### Early Access Permission types
```

Comment:
```suggestion
```
You probably don't need a new section for these permissions. Just append them to the existing table and tag the permission with the EA flag.